src/app/public/modules/ag-grid/ag-grid.service.ts

```diff
@@ -297,6 +339,43 @@ export class SkyAgGridService implements OnDestroy {
       suppressDragLeaveHidesColumns: true
     };
 
+    defaultSkyGridOptions.columnTypes[SkyCellType.CurrencyValidator] = {
+      ...defaultSkyGridOptions.columnTypes[SkyCellType.Validator],
+      ...defaultSkyGridOptions.columnTypes[SkyCellType.Currency],
+      cellClassRules: {
+        ...defaultSkyGridOptions.columnTypes[SkyCellType.Validator].cellClassRules,
+        ...defaultSkyGridOptions.columnTypes[SkyCellType.Currency].cellClassRules
+      },
+      cellRendererFramework: undefined,
+      cellRendererSelector: getValidatorCellRendererSelector('sky-ag-grid-cell-renderer-currency-validator'),
+      cellRendererParams: {
+        /*istanbul ignore next*/
+        validator: (value: any) => {
+          // TODO: This needs to be tightened up around which currencies to allow and more specific matching.
+          /*istanbul ignore next*/
+          return !!`${value || ''}`.match(/^[^0-9]*(\d+[,.]?)+\d*[^0-9]*$/);
+        },
+        validatorMessage: 'Please enter a valid currency'
+      }
+    };
+
+    defaultSkyGridOptions.columnTypes[SkyCellType.NumberValidator] = {
+      ...defaultSkyGridOptions.columnTypes[SkyCellType.Validator],
+      ...defaultSkyGridOptions.columnTypes[SkyCellType.Number],
+      cellClassRules: {
+        ...defaultSkyGridOptions.columnTypes[SkyCellType.Validator].cellClassRules,
+        ...defaultSkyGridOptions.columnTypes[SkyCellType.Number].cellClassRules
+      },
+      cellRendererParams: {
+        /*istanbul ignore next*/
+        validator: (value: any) => {
+          /*istanbul ignore next*/
+          return !!value && !isNaN(parseFloat(value));
```

Comment:
Same question regarding this being able to be overwritten.

Reply:
Yes, the validator message can be overridden.